services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/DigitalCovidCertificateFeignClient.java

```diff
@@ -58,14 +58,22 @@
   @Timed
   @GetMapping(value = "${services.distribution.digital-green-certificate.client.bn-rules-path}")
   ResponseEntity<List<BusinessRuleItem>> getBoosterNotificationRules();
+
   /**
    * HTTP GET to return a specific business rule based on its country and hash.
    *
    * @param country - business rule country code.
    * @param hash - business rule hash
    */
-
   @Timed
   @GetMapping(value = "${services.distribution.digital-green-certificate.client.rules-path}/{country}/{hash}")
   ResponseEntity<BusinessRule> getCountryRule(@PathVariable String country, @PathVariable String hash);
+
+  /**
+   * HTTP GET to return a specific business rule based on its country and hash.
+   * @param hash - business rule hash
+   */
+  @Timed
+  @GetMapping(value = "${services.distribution.digital-green-certificate.client.bn-rules-path}/{hash}")
+  ResponseEntity<BusinessRule> getBnRule(@PathVariable String hash);
```

Comment:
I would recommend not to abreviate this method names for consistency. No other method is abreviated in the Feign client. Also would rename all the other methods further down in the code which uses "Bn". It is just not clear enough.